contrib/windows/msys_build.sh

```diff
@@ -203,7 +203,7 @@ fi
 echo 'FORCE_ASSERTIONS = 1' >> Make.user
 
 cat Make.user
-make -j3 VERBOSE=1 install
+make -j3 VERBOSE=1
```

Comment:
may as well keep the install, it takes an extra couple minutes but we have the capacity and it's worth covering